hw/ip/lc_ctrl/doc/_index.md

```diff
@@ -195,41 +200,220 @@ This ensures that during specific states (RAW, TEST_LOCKED, SCRAP, INVALID) it i
+
+For more a list of the collateral in Flash and OTP and an explanation of how that collateral is affected by these signals, see the [OTP collateral]({{< relref "#otp-collateral" >}}) and [flash collateral]({{< relref "#flash-collateral" >}}) sections.
+
+#### SEED_HW_RD_EN
+
+The SEED_HW_RD_EN signal controls whether the owner and creator root keys can be accessed by hardware.
+This signal is dependent on the personalization state of the device and will only be enabled if the device has been personalized (i.e., when the OTP partition holding the root key has been locked down).
+
+#### ISO_PART_SW_RD_EN and ISO_PART_SW_WR_EN
+
+These signals control whether the isolated flash partition holding additional manufacturing details can be accessed.
+The isolated partition is both read and writable during the DEV / PROD / PROD_END / RMA states.
+In all other states it is inaccessible, except during the TEST_UNLOCKED state where the partition is write-only.
+This construction allows to write a value to that partitition and keep it secret before advancing into any of the production states.
+
+
+## OTP Collateral
+
+The following is a list of all life cycle related collateral stored in OTP.
+Most collateral also contain associated metadata to indicate when the collateral is restricted from further software access, see [accessibility summary]({{< relref "#otp-accessibility-summary-and-impact-of-provision_en" >}}) for more details.
+Since not all collateral is consumed by the life cycle controller, the consuming agent is also shown.
+
+{{< snippet "hw/ip/lc_ctrl/doc/lc_ctrl_otp_collateral.md" >}}
+
+The TOKENs and KEYS are considered secret data and are stored in [wrapped format]({{< relref "#conditional-transitions">}}).
+Before use, the secrets are unwrapped.
+
+The SECRET0_DIGEST and SECRET2_DIGEST are the digest values computed over the secret partitions in OTP holding the tokens and root keys.
+As described in more detail in the [OTP controller specification]({{< relref "hw/ip/otp_ctrl/doc/_index.md#direct-access-memory-map">}}), these digests have a non-zero value once the partition has been provisioned and write/read access has been locked.
+
+### ID State of the Device
+
+If the SECRET2_DIGEST is zero, the device is considered to have "blank" ID state, in which case the CREATOR_ROOT_KEY_* (in OTP) and CREATOR_DIV_KEY (in FLASH) can be written by software.
+All consumers of these keys are supplied with an invalid value.
+
+If the SECRET2_DIGEST has a nonzero value, the device is considered "creator personalized", and the CREATOR_ROOT_KEY and CREATOR_DIV_KEY are no longer accessible to software.
+Actual values are supplied to the consumers.
+If SECRET2_DIGEST has a nonzero value, the CREATOR_SEED_SW_RW_EN signal will be disabled in PROD, PROD_END and RMA states.
```

Comment:
is the last part regarding RMA states still true?

Reply:
yes I need to change this to PROD, PROD_END and DEV states